Proteomics/Protein/ProteoformLevelClassifier.cs

```diff
@@ -0,0 +1,223 @@
+﻿using Proteomics.ProteolyticDigestion;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Proteomics
+{
+    public static class ProteoformLevelClassifier
+    {
+
+        /// <summary>
+        /// All input strings are delimited with "|"
+        /// PTMs are annotated with []
+        /// </summary>
+        /// <param name="fullSequenceString">All possible sequences (with modifications) for this PrSM</param>
+        /// <param name="geneString">All possible genes for this PrSM</param>
+        /// <returns></returns>
+        public static string ClassifyPrSM(string fullSequenceString, string geneString)
```

Comment:
Nice! You could consider adding something like this to https://github.com/topdownproteomics/sdk, too.

Reply:
Great idea!